lib/plugins/aws/customResources/resources/eventBridge/lib/utils.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+const crypto = require('crypto');
+
 function getEventBusName(eventBus) {
   if (eventBus && eventBus.startsWith('arn')) {
     return eventBus.slice(eventBus.indexOf('/') + 1);
   }
   return eventBus;
 }
 
+function getEventBusTargetId(ruleName) {
+  const targetIdSuffix = 'target';
+  let targetId = `${ruleName}-${targetIdSuffix}`;
+  if (targetId.length > 64) {
+    // Target ids cannot be longer than 64.
+    // Temporary solution until we have https://github.com/serverless/serverless/issues/6598
```

Comment:
I think #6598 is not directly related to that issue (it covers very general improvements to how custom resources are deployed by the framework)